changelog/6466.improvement.rst

```diff
@@ -0,0 +1,6 @@
+[Response selector](retrieval-actions.mdx) is now trained on retrieval intent labels by default instead of actual response text.
```

Comment:
This file should be `changelog/6466.improvement.md` and you no longer need the double backticking for code formatting!